shared/lib_pvshade.h

```diff
@@ -140,8 +156,8 @@ bool ss_exec(
 	double albedo,		// used to calculate reduced relected irradiance
 	bool trackmode,		// 0 for fixed tilt, 1 for one-axis tracking
 	bool linear,		// 0 for non-linear shading (C. Deline's full algorithm), 1 to stop at linear shading
-	double shade_frac_1x,	// geometric calculation of the fraction of one-axis row that is shaded (0-1), not used if fixed tilt 
-	
-	ssoutputs &outputs);
+	double shade_frac_1x,	// geometric calculation of the fraction of one-axis row that is shaded (0-1), not used if fixed tilt
+    ssSkyDiff &skydiffs,
```

Comment:
same comment as above- include detailed comment and consider less confusing variable name